build-system/tasks/create-golden-css/css/main.css

```diff
@@ -146,49 +146,42 @@ html.i-amphtml-ios-embed.i-amphtml-ios-overscroll > #i-amphtml-wrapper {
   border-top: 1px solid transparent !important;
 }
 
-
 .i-amphtml-element {
   display: inline-block;
 }
 
 .i-amphtml-layout-nodisplay,
-[layout=nodisplay]:not(.i-amphtml-layout-nodisplay)
-{
+[layout='nodisplay']:not(.i-amphtml-layout-nodisplay) {
```

Comment:
I'm a bit surprised to see these changes in this PR, but they are non-breaking a look like prettier reformatting.

Reply:
Reverted this change.